Exercise_2.py

```diff
@@ -2,15 +2,25 @@
   
 # give you explanation for the approach
 def partition(arr,low,high):
-  
-  
     #write your code here
-  
+    l = low-1
+    pivot = arr[high]
+    for k in range(l, high):
```

Comment:
The iteration should be from "range(low, high)" since L for the first iteration will be -1.